ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/utils/FontFamilyXMLParser.kt

```diff
@@ -0,0 +1,131 @@
+package com.revenuecat.purchases.ui.revenuecatui.utils
+
+import android.content.res.XmlResourceParser
+import androidx.annotation.VisibleForTesting
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import com.revenuecat.purchases.ui.revenuecatui.helpers.Logger
+import org.xmlpull.v1.XmlPullParser
+
+@Suppress("NestedBlockDepth")
+internal object FontFamilyXMLParser {
```

Comment:
[Kotlin convention](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names) nit: 
```suggestion
internal object FontFamilyXmlParser {
```